runtime/src/main/java/io/quarkus/oidc/proxy/runtime/OidcProxy.java

```diff
@@ -511,6 +513,7 @@ private String buildUri(RoutingContext context, String path) {
                 : context.request().scheme();
         return new StringBuilder(scheme).append("://")
                 .append(authority)
+                .append(httpRootPath)
```

Comment:
`Router` will prefix paths with the `quarkus.http.root-path` value and we can remove the `httpRootPath` from the setup.

But here we need to manually prefix paths with `httpRootPath` because this is used while generating the URLs in `.well-known/openid-configuration`, not using the Router. I'm not sure if there's a way to get these routes dynamically.

Reply:
@thiagoelg OK, thanks